serving/src/main/java/ai/djl/serving/models/ModelManager.java

```diff
@@ -388,6 +388,90 @@ public CompletableFuture<FullHttpResponse> workerStatus() {
                 });
     }
 
+    /**
+     * Sends model server health status to client.
+     *
+     * @param modelName of certain modelName
+     * @param modelVersion certain version of param
+     * @return completableFuture with eventually result in the future after async execution
+     */
+    public CompletableFuture<FullHttpResponse> modelStatus(String modelName, String modelVersion) {
```

Comment:
You should reuse workerStatus() and return a proper WorkerStatus object instead of FullHttpResponse, and convert to FullHttpResponse in handler, so you can reuse it for both KServer and DJL case.